sections/06_discussion.md

```diff
@@ -49,6 +49,86 @@ with only a couple GPUs.*
 *Some of this is also outlined in the Categorize section.  We can decide where
 it best fits.*
 
+Efficiently scaling deep learning is challenging, and there is a high
+computational cost (e.g., time, memory, energy) associated with training neural
+networks and using them for classification. For these reasons, neural networks
+have only recently found widespread use [@tag:Schmidhuber2014_dnn_overview].
+For biologists, such problems are further complicated by the immense size of
+most biological datasets.
```

Comment:
I would argue many of the datasets used in our biomedical examples are not that large, especially compared to some image or speech datasets.  Should we highlight more specific examples of the problems where the size of the data on disk is a limiting factor (perhaps imaging, applications with (epi)genomic input data, etc.)